### DISCUSSION TITLE
Table data change

### DISCUSSION THREAD

**Discussion by Vik1ang** [ant-design-blazor/ant-design-blazor#2808] [Q&A] [answered]
I wanna know how to reload the table while data dynamically  change?

**I try to use `table.ReloadData()`, but it will cancel all table state like filter or sort ( it will init filter sort).**

Is Antd blazor has another better solution for this case?

**Comment by ElderJames** [comment, marked as answer]
Please try `Table.ReloadData(queryModel)` to resotre the query state when reloading data.

https://ant-design-blazor.gitee.io/zh-CN/components/table#components-table-demo-RestoreQueryState

**Reply by Vik1ang** [reply]
Thanks so much!

**Reply by ElderJames** [reply]
Welcome join our commuinty in WeChat.

**Reply by Vik1ang** [reply]
> Please try `Table.ReloadData(queryModel)` to resotre the query state when reloading data.
> 
> [ant-design-blazor.gitee.io/zh-CN/components/table#components-table-demo-RestoreQueryState](https://ant-design-blazor.gitee.io/zh-CN/components/table#components-table-demo-RestoreQueryState)

I wanna ask another question, how to avoid collapse table after using `Table.ReloadDate()`, espically i dont wanna use ExpandDefault?

**Reply by Vik1ang** [reply]
> Welcome join our commuinty in WeChat.

Hi, I didnt find Wechat link for commuinty but there is Dingding link in the repo, where i can find the link for  wechat community?

**Reply by ElderJames** [reply]
Add my WeChat JamesYeungMVP first.
